controllers/object.js

```diff
@@ -196,10 +196,14 @@ const activate = function(objectID, callback) {
 };
 
 const setVisualization = function(objectID, vis, callback) {
-    utilities.getObject(objects, objectID).visualization = vis;
-    console.log(vis, objectID);
-    utilities.writeObjectToFile(objects, objectID, objectsPath, globalVariables.saveToDisk);
-    callback(200, 'ok');
+    try {
```

Comment:
It would be a bit better to write this as
```
let object = utilities.getObject(...);
if (!object) {
  callback with 404
  return
}
rest of function
```
since right now if writeObjectToFile throws an error (like if the disk is full or there aren't permissions for objectsPath) it will be misreported as a 404